libs/stx/inc/public/tfc/stx/glaze_meta.hpp

```diff
@@ -53,6 +52,23 @@ struct glz::meta<std::chrono::duration<rep_t, period_t>> {
     tfc::stx::string_view_join_v<prefix, glz::name_v<rep_t>, separator, glz::name_v<period_t, true>, postfix>
   };
 };
+template <>
+struct glz::meta<std::chrono::system_clock> {
+  static constexpr std::string_view name{ "std::chrono::system_clock" };
+};
+template <>
+struct glz::meta<std::chrono::steady_clock> {
+  static constexpr std::string_view name{ "std::chrono::steady_clock" };
+};
+template <typename clock_t, typename duration_t>
+struct glz::meta<std::chrono::time_point<clock_t, duration_t>> {
+  static constexpr std::string_view prefix{ "std::chrono::time_point<" };
+  static constexpr std::string_view postfix{ ">" };
+  static constexpr std::string_view separator{ "," };
+  static constexpr auto name{
+    tfc::stx::string_view_join_v<prefix, glz::name_v<clock_t, true>, separator, glz::name_v<duration_t, true>, postfix>
+  };
+};
```

Comment:
This creates a non standard time representation that requires custom parsing in each platform
wanting to read it.
I don't see a use case for this level of precision being sent between systems.

as a milliseconds since 1 Jan 1970 I would not be opposed or simply seconds since 1970
this also loses all timezone information.

What does the json from this look like? is it just a number or is the duration period encoded as well?

does the receiving party need the json schema to realize the resolution from parsing the name?

Reply:
> This creates a non standard time representation that requires custom parsing in each platform
wanting to read it.
I don't see a use case for this level of precision being sent between systems.

Now time_point of milliseconds is represented in iso8601 format.

> as a milliseconds since 1 Jan 1970 I would not be opposed or simply seconds since 1970
this also loses all timezone information.

timezone info is dependent on system_clock so this would not work robustly across different hosts

> What does the json from this look like? is it just a number or is the duration period encoded as well?

Time_point in micro or nanoseconds are represented as number, resulting in no truncation.

does the receiving party need the json schema to realize the resolution from parsing the name?

Ideally the receiving party knows the type

Reply:
This just looks like something we don't want to be doing. What use case are we solving with this?

Reply:
removed